API/Controllers/ProjectController.cs

```diff
@@ -792,6 +790,245 @@ public async Task<IActionResult> UnlikeProject(int projectId)
+                    Instance = "11deede6-c76c-42cd-ac50-97743e3cff2a"
+                };
+                return NotFound(problemDetails);
+            }
+
+            if(!await projectService.ProjectExistsAsync(projectId))
+            {
+                ProblemDetails problemDetails = new ProblemDetails
+                {
+                    Title = "Failed setting the status of the project.",
+                    Detail = "The project send in the request could not be found in the database.",
+                    Instance = "4a73928f-827f-44a5-b508-e6a8c73c1717"
+                };
+                return NotFound(problemDetails);
+            }
+
+            if(!await institutionService.InstitutionExistsAsync(institutionId))
+            {
+                ProblemDetails problemDetails = new ProblemDetails
+                {
+                    Title = "Failed getting the institution.",
+                    Detail = "The institution send in the request could not be found in the database.",
+                    Instance = "01cdc6f4-42aa-4394-bbc8-2576e4f99498"
+                };
+                return NotFound(problemDetails);
+            }
+
+            if(projectInstitutionService.InstitutionIsLinkedToProject(projectId, institutionId))
+            {
+                ProblemDetails problemDetails = new ProblemDetails
+                {
+                    Title = "Institution is already linked to project.",
+                    Detail = "The institution cannot be linked to the project because the institution is already part of the project.",
+                    Instance = "01b26993-0c9e-4890-a7fe-f0f39450e616"
+                };
+                return Conflict(problemDetails);
+            }
+
+            ProjectInstitution projectInstitution = new ProjectInstitution(projectId, institutionId);
+            await projectInstitutionService.AddAsync(projectInstitution);
+            projectInstitutionService.Save();
+
+            //Maybe a bit wasteful to get the entire project and institution but its fast enough for now.
+            projectInstitution.Project = await projectService.FindAsync(projectId);
+            projectInstitution.Institution = await institutionService.FindAsync(institutionId);
+
+            return Created(nameof(LinkInstitutionToProjectAsync), mapper.Map<ProjectInstitution, ProjectInstitutionResourceResult>(projectInstitution));
+        }
+
+        /// <summary>
+        /// Links given project and institution. This function is admin only!
+        /// </summary>
+        /// <param name="projectId">The project identifier</param>
+        /// <param name="institutionId">The institution identifier</param>
+        /// <returns>
+        ///     StatusCode 201 If success,
+        ///     StatusCode 404 If the project, institution or current user couldn't be found.
+        ///     StatusCode 409 If the project is not yet linked to the institution.
+        /// </returns>
```

Comment:
Try to use these response comments because Swagger uses it for documentation.
https://github.com/DigitalExcellence/dex-backend/blob/develop/API/Controllers/CallToActionOptionController.cs#L89